docs/snippets/common/storybook-main-register-presets-managerentry.js.mdx

```diff
@@ -3,7 +3,7 @@
 
 module.exports = {
   addons: [
-    '@storybook/addon-storysource/register', // A managerEntry registered here, in this case from the storysource addon.
+    '@storybook/addon-storysource/manager', // A managerEntry registered here, in this case from the storysource addon.
```

Comment:
Do we want to document this?

Reply:
We spoke about removing all references to the storysource-addon, I'd be in favor of doing that.